source/includes/kubernetes/_k8_secrets.md

```diff
@@ -18,10 +18,10 @@ curl -X GET \
     {
       "id": "default-token-xxxmt/default",
       "apiVersion": "v1",
-      "data": {
-            "ca.crt": [],
-            "namespace": [],
-            "token": []
+      "encodedData": {
+        "ca.crt": "LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUN3akNDQWFxZ0F3SUJBZ0lCQURBTkJna3Foa2lHOXcwQkFRc0ZBREFTTVJBd0RnWURWUVFERXdkcmRXSmwKTFdOaE1CNFhEVEl3TURZeE1URTJNVGMxTjFvWERUTXdNRFl3T1RFMk1UYzFOMW93RWpFUU1BNEdBMVVFQXhNSAphM1ZpWlMxallUQ0NBU0l3RFFZSktvWklodmNOQVFFQkJRQURnZ0VQQURDQ0FRb0NnZ0VCQUx5K2ZkWG9NMi80CjhWallYM0VJMHV5b2Y4clJjbkU3RjFDclFZc0RkNmNaV3IwbGpDemtXclk4QThGZUhHT3NYc3Q4V2wvNDltenMKWXVZYks1QXkyeWFNcWp3NnNZTnprMXIrZTZDUjFmaGdRNHpMNDREU1FsR2xVOHk3N2VpOHdOTVR6d09FOXIxMgpJdlFKVGNkS1llclFxNjZ5YTJuSklrNHo4Y2sxc2MzQVZmK1JSNXRaNWtGUUptRis5dHpickNFd2IxWUQ5RkdECjczQVNKSG9YK0wxRWtEYWRKU2NJYmdtWklxUlNwdkJBZHdGdlJLRnhId2VkcnE0b0JwZ0Z4RkEwMEJlM2dyMU0KYlhmL1p1VTZPRU1YbmY3VmNCT2g4QTgvbUYwT0htL3d4WXJ3bHVqc3oyVXlRUVNkTlZueHVpMmJrSm00UlUvNQpDVnEycElwY2o5TUNBd0VBQWFNak1DRXdEZ1lEVlIwUEFRSC9CQVFEQWdLa01BOEdBMVVkRXdFQi93UUZNQU1CCkFmOHdEUVlKS29aSWh2Y05BUUVMQlFBRGdnRUJBRlJDdjh6cnAwNm1LRC9DWGVjVHpXdFhKbkVJR3dRY2F6eC8KSzZYSDgwaFFVbllKcDBBZkNlNlp1SjlMc2ZtalZhVmRSTjUyaGZmZ3RHZjU1TFkvUUMvdnE2Wkp6VklDVEdSMQpZclczY3Vld0k0b1hmUGJ0WWhUZks0TEk4M0p3dGJDYThTZTdNMFNsZnl4OWN1WlIxT2xiZ3kxbTF4MmlId1VlCklnblA0YzlnSG1aUmhFR2xnTkp1cDNVbVRkUi9KTnliMGdCVk9VdExldkV4cUpvNldhaHFyL1FnNmhZRHpsZzAKT0N2cGYwSFRLQUJRZklsldfikhbEU3BSRVhBYlNFbFR3aFpQRFJNSmNvYTNzRDhsam9JS1NFS0FvNkJKenFaNHg1Wk01ZwppLzZFS2EwRzU3enVMbk1vczhKU3FKUHhLU25SZ2ExckR1M0g2S2Nway8zNFVCUGZMQzg9Ci0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K",
+        "namespace": "a3ViZS1zeXN0ZW0=",
```

Comment:
It might make sense to give a more readable example for namepsace

Reply:
It's also encoded 😛

Reply:
😓 🙏